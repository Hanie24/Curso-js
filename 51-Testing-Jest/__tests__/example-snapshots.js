const obj = {
  name: "Hanie",
  balance: 500,
  type: "premium",
};

describe("Object testing", () => {
  test("The person is Hanie", () => {
    expect(obj).toMatchSnapshot("Hanie");
  });
});
