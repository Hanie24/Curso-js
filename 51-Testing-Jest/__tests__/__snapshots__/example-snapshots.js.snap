// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Object testing The person is Hanie: Hanie 1`] = `
{
  "balance": 500,
  "name": "Hanie",
  "type": "premium",
}
`;
